{"ast":null,"code":"'use strict';\n\nvar AxiosError = require('../core/AxiosError');\nvar utils = require('../utils');\n\n/**\r\n * A `CanceledError` is an object that is thrown when an operation is canceled.\r\n *\r\n * @class\r\n * @param {string=} message The message.\r\n */\nfunction CanceledError(message) {\n  // eslint-disable-next-line no-eq-null,eqeqeq\n  AxiosError.call(this, message == null ? 'canceled' : message, AxiosError.ERR_CANCELED);\n  this.name = 'CanceledError';\n}\nutils.inherits(CanceledError, AxiosError, {\n  __CANCEL__: true\n});\nmodule.exports = CanceledError;","map":{"version":3,"names":["AxiosError","require","utils","CanceledError","message","call","ERR_CANCELED","name","inherits","__CANCEL__","module","exports"],"sources":["C:/Users/dlfrn/Downloads/myToyProj/toyProject/FrontEnd/alcohol-app/node_modules/axios/lib/cancel/CanceledError.js"],"sourcesContent":["'use strict';\r\n\r\nvar AxiosError = require('../core/AxiosError');\r\nvar utils = require('../utils');\r\n\r\n/**\r\n * A `CanceledError` is an object that is thrown when an operation is canceled.\r\n *\r\n * @class\r\n * @param {string=} message The message.\r\n */\r\nfunction CanceledError(message) {\r\n  // eslint-disable-next-line no-eq-null,eqeqeq\r\n  AxiosError.call(this, message == null ? 'canceled' : message, AxiosError.ERR_CANCELED);\r\n  this.name = 'CanceledError';\r\n}\r\n\r\nutils.inherits(CanceledError, AxiosError, {\r\n  __CANCEL__: true\r\n});\r\n\r\nmodule.exports = CanceledError;\r\n"],"mappings":"AAAA,YAAY;;AAEZ,IAAIA,UAAU,GAAGC,OAAO,CAAC,oBAAoB,CAAC;AAC9C,IAAIC,KAAK,GAAGD,OAAO,CAAC,UAAU,CAAC;;AAE/B;AACA;AACA;AACA;AACA;AACA;AACA,SAASE,aAAaA,CAACC,OAAO,EAAE;EAC9B;EACAJ,UAAU,CAACK,IAAI,CAAC,IAAI,EAAED,OAAO,IAAI,IAAI,GAAG,UAAU,GAAGA,OAAO,EAAEJ,UAAU,CAACM,YAAY,CAAC;EACtF,IAAI,CAACC,IAAI,GAAG,eAAe;AAC7B;AAEAL,KAAK,CAACM,QAAQ,CAACL,aAAa,EAAEH,UAAU,EAAE;EACxCS,UAAU,EAAE;AACd,CAAC,CAAC;AAEFC,MAAM,CAACC,OAAO,GAAGR,aAAa","ignoreList":[]},"metadata":{},"sourceType":"script","externalDependencies":[]}